src/workloads/docs/RollingCollections.yml

```diff
@@ -1,7 +1,12 @@
 SchemaVersion: 2018-07-01
 Owner: Storage Engines
 Description: |
-  TODO: TIG-3319
+  A workload to test/document the RollingCollections actor which has two modes of operation a
+  a "Setup" operation which creates CollectionWindowSize collections and populates them with
```

Comment:
An extra "a" -- "two modes of operation a a Setup...".  Maybe a comment or period after "operation".